README.md

```
@@ -1,36 +1,20 @@
+
+Hatch Tamagotchis 		POST http://localhost:8080/api/tamagotchis
+Get All Tamagotchis 		GET http://localhost:8080/api/tamagotchis
+Get Tamagotchi by id 		GET http://localhost:8080/api/tamagotchis/id
+
+Feed 					PUT http://localhost:8080/api/tamagotchis/id/feed
+Play 					PUT http://localhost:8080/api/tamagotchis/id/play
+Sleep 					PUT http://localhost:8080/api/tamagotchis/id/sleep
+Wake 					PUT http://localhost:8080/api/tamagotchis/id/wake
+Clean 					PUT http://localhost:8080/api/tamagotchis/id/clean
+Medicine 				PUT http://localhost:8080/api/tamagotchis/id/medicine
+Release					ERASE http://localhost:8080/api/tamagotchis/id
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 8

(MD010, no-hard-tabs)